src/polynomial.jl

```diff
@@ -0,0 +1,49 @@
+# function Polynomial{C,E}(vec::AbstractVector, mat::AbstractMatrix) where {C <: Union{CxxWrap.CxxLong,VecOrMat_eltypes}, E <: Union{CxxWrap.CxxLong, Int64}}
+#     v = Vector{C}(vec)
+#     m = Matrix{E}(mat)
+#     return Polynomial{to_cxx_type(C),to_cxx_type(E)}(v, m)
+# end
+#
+# function Polynomial{C,E}(p::Polynomial) where {C,E}
+#     v = coefficients_as_vector(p)
+#     m = monomials_as_matrix(p)
+#     return Polynomial{C,E}(v, m)
+# end
+#
+# function Polynomial{C,E}(n::Number, d::Base.Integer) where {C,E}
+#     m = zeros(1,d)
+#     return Polynomial{C,E}([n],m)
+# end
+#
+# Polynomial(vec::AbstractVector{C}, mat::AbstractMatrix{E}) where {C,E} = Polynomial{promote_to_pm_type(Vector,C),Int64}(vec, mat)
```

Comment:
please shorten/break those lines so that they fit into reasonable (make it 75-90) line width